features/bevragen/autorisatie/autorisatie.feature

```diff
@@ -380,110 +287,55 @@ Functionaliteit: autorisatie voor het gebruik van de API
-      | type     | https://datatracker.ietf.org/doc/html/rfc7231#section-6.5.3             |
-      | title    | U bent niet geautoriseerd voor één of meerdere opgegeven field waarden. |
-      | status   | 403                                                                     |
-      | code     | unauthorizedField                                                       |
-      | instance | /haalcentraal/api/brp/personen                                          |
-
-    @fout-case
-    Scenario: Gemeente raadpleegt een inwoner van een andere gemeente met de gemeenteVanInschrijving parameter en fields vraagt om partners.geslacht (50410) dat niet in de autorisatie zit
-      Gegeven de afnemer met indicatie '000008' heeft de volgende 'autorisatie' gegevens
-      | Rubrieknummer ad hoc (35.95.60) | Medium ad hoc (35.95.67) | Datum ingang (35.99.98) |
-      | 10120 50240 50610               | N                        | 20201128                |
-      En de geauthenticeerde consumer heeft de volgende 'claim' gegevens
-      | naam         | waarde |
-      | afnemerID    | 000008 |
-      | gemeenteCode | 0800   |
-      En de persoon met burgerservicenummer '000000024' heeft de volgende 'verblijfplaats' gegevens
-      | gemeente van inschrijving (09.10) |
-      | 0599                              |
-      En de 'verblijfplaats' heeft de volgende 'adres' gegevens
-      | gemeentecode (92.10) |
-      | 0599                 |
-      En de persoon heeft een 'partner' met de volgende gegevens
-      | geslachtsnaam (02.40) | geslachtsaanduiding (04.10) |
-      | Groenen               | M                           |
-      Als gba personen wordt gezocht met de volgende parameters
-      | naam                    | waarde                          |
-      | type                    | RaadpleegMetBurgerservicenummer |
-      | burgerservicenummer     | 000000024                       |
-      | gemeenteVanInschrijving | 0599                            |
-      | fields                  | partners.geslacht               |
-      Dan heeft de response een object met de volgende gegevens
-      | naam     | waarde                                                                  |
-      | type     | https://datatracker.ietf.org/doc/html/rfc7231#section-6.5.3             |
-      | title    | U bent niet geautoriseerd voor één of meerdere opgegeven field waarden. |
-      | status   | 403                                                                     |
-      | code     | unauthorizedField                                                       |
-      | instance | /haalcentraal/api/brp/personen                                          |
+      Voorbeelden:
+      | gemeente van inschrijving | omschrijving                    |
+      | 0800                      | eigen inwoner                   |
+      | 0530                      | inwoner van een andere gemeente |
```

Comment:
dit geeft geen personen terug omdat de betreffende persoon in een ander gemeente staat ingeschreven. Of de gemeenteVanInschrijving moet ook gebruik maken van de 'gemeente van inschrijving' voorbeeld waarde